QuadradiusR/scripts/tile.gd

```diff
@@ -23,7 +23,8 @@ const ELEVATION_OFFSET = -50
 var tile_id: String
 var is_steppable = true
 var tile_pos: Vector3  # Z - elevation, from -2 to +2, total 5 levels
-
+var orb_id: String
+var has_orb = false
```

Comment:
I think this field is not needed - when `orb_id == nil` it means there is no orb on it, when there is `orb_id != nil`. We can instead have a method `has_orb()` in tile making that check. Keeping track of two fields is more bug-generating 😸

Reply:
correct, I'll fix this

- [ ] Remove redundant ```has_orb``` field